src/client/src/containers/AzureFunctionsModal/messages.ts

```diff
@@ -93,5 +93,17 @@ export const messages = defineMessages({
   numFunctionsSubLabel: {
     id: "azureFunctionsModal.numFunctionsSubLabel",
     defaultMessage: "Select the number of endpoints you want for your web app"
+  },
+  chooseExisting: {
+    id: "azureFunctionsModal.chooseExisting",
+    defaultMessage: "Choose existing"
```

Comment:
Can this be more specific? e.g. chooseExistingResourceGroup, with default message "Choose from existing resource groups in the subscription"

Reply:
This is for the radio button though, I thought we agreed on this :(

Reply:
I will change the variable name though